engine/src/exec-interface-object.cpp

```diff
@@ -4657,7 +4660,11 @@ void MCObject::GetRevAvailableVariables(MCExecContext& ctxt, MCNameRef p_key, MC
     }
     
     // The handler code must be the first char of the string
-    const char_t t_handler_code = MCStringGetNativeCharAtIndex(t_key, 0);
+    char_t t_handler_code = MCStringGetNativeCharAtIndex(t_key, 0);
+    
+    // private handlers are P{F|M}
+    if (t_handler_code == 'P')
+        t_handler_code = MCStringGetNativeCharAtIndex(t_key, 1);
```

Comment:
Ah, I forgot about that bit... I'll remove from this PR as it's unrelated it was just because the `revAvailableHandlers` returns PF and PM for private handlers. I don't think we will use `revAvailableHandlers` all that much once we have `revScriptDescription` so probably not worth changing.